File: php/6kyu - G.php
<?php

//Link -> https://www.codewars.com/kata/5287e858c6b5a9678200083c

function narcissistic(int $value): bool {
    $valueString = (string)$value;
    $stringSize = strlen($valueString);
    $bigNumber = 0;

    for($i = 0; $i < $stringSize; $i++){
        $number = (int)$valueString[$i];
        $bigNumber += pow($number, $stringSize);
    }

    switch($bigNumber){
        case $value:
          return true;
        default:
          return false;
    }
}

?>

File: python/6kyu - G.py
# Link -> https://www.codewars.com/kata/5679aa472b8f57fb8c000047

def find_even_index(arr):
    # Size array
    i = len(arr)

    # Default
    N = -1

    # Slice array
    for i in range(0, i):
        a = arr[0:i]
        b = arr[i + 1:]

        # Sums
        if sum(a) == sum(b):
            N = i
            break

    return N
